trainer/app_code/model_files.py

```diff
@@ -22,19 +22,22 @@ def get_all_weightfiles(training_path: Path) -> List[Path]:
     return weightfiles
 
 
-def _epoch_from_weightfile(weightfile: Path) -> int:
-    number = weightfile.name[5:-3]
-    if number == '':
+def epoch_from_weightfile(weightfile: Path) -> int:
+    try:
+        number = weightfile.name[5:-3]
+        if number == '':
+            return 0
+        return int(number)
+    except ValueError:
         return 0
```

Comment:
When could this happen?
We will most likely remove it in `delete_older_epochs`. Is this what we want?

Reply:
What do you mean by 'it' ???

Reply:
the epoch weight file